### DISCUSSION TITLE
Get sticky: state

### DISCUSSION THREAD

**Discussion by gotyourchalk** [tailwindlabs/tailwindcss#2265] [Ideas]
Can there be a feature that when sticky: do something?  Kind of like the hover: state.

I have sticky cards but eventually, the shadows begin to stack one upon the other.

clase="shadow-md sticky sticky:shadow-none"

**Comment by brandonpittman** [comment]
@gotyourchalk What would the original CSS for checking sticky even be? Like group-hover but group-sticky?
